frontend/src/components/UserDetail/PastRides.tsx

```diff
@@ -9,17 +9,10 @@ type pastRideProps = {
 };
 
 const PastRides = ({ isStudent, rides }: pastRideProps) => {
-  function renderTableHeader() {
-    return (
-      <tr>
-        <th className={styles.tableHeader}>{isStudent ? 'Date' : 'Name'}</th>
-        <th className={styles.tableHeader}>{isStudent ? 'Time' : 'Date'}</th>
-        <th className={styles.tableHeader}>Pickup Location</th>
-        <th className={styles.tableHeader}>Dropoff Location</th>
-        <th className={styles.tableHeader}>Needs</th>
-      </tr>
-    );
-  }
+  const colSizes = [2, 2, 2, 2, 2];
+  const header1 = isStudent ? 'Date' : 'Name';
+  const header2 = isStudent ? 'Time' : 'Date';
+  const headers = [header1, header2, 'Pickup Location', 'Dropoff Location', 'Needs'];
```

Comment:
It would be nice to use a more descriptive name rather than `header1`, `header2`.
Alternatively, you could do something like this:
```
const studentHeaders = ['Date', 'Time', ...];
const employeeHeaders = ['Name', 'Date', ...];
```